docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc

```diff
@@ -27,18 +27,18 @@ include::{includes}/prerequisites.adoc[]
 
 == Architecture
 
-In this example, we build a very simple web application with a single page:
+In this example, we build a simple web application with a single page:
 
 * `/index.html`
 
-This page is protected and can only be accessed by authenticated users.
+This page is protected and only authenticated users can access the page.
 
 == Solution
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
-However, you can go right to the completed example.
+Follow the instructions in the next sections and create the application step by step.
+Alternatively, you can go right to the completed example.
 
-Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+Clone the Git repository by using `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
```

Comment:
```suggestion
Clone the Git repository by running a `git clone {quickstarts-clone-url}` command, or download an {quickstarts-archive-url}[archive].
```

I'm not sure it is totally correct from the language perspective, but `by using` reads a little bit unusual

Reply:
Good point. Thank you. Updated per your suggestion.